src/App.js

```diff
@@ -23,20 +23,20 @@ class App extends Component {
   // }
 
   auth() {
-    // window.location.href = "/auth/google";
-    let headers = new Headers();
+    //window.location.href = "/auth/google";
```

Comment:
Non-blocking: Trimming out some of the extra comments so we keep things tidy would be ace 😉